kit.yml

```diff
@@ -123,6 +14,21 @@ credentials:
     grafana:
       password:   random 64
       secret_key: random 64
+    alertmanager:
+      password: random 64
+
+  http-auth:
+    nginx/alert_manager:
+      password: random 64
+    nginx/prometheus:
+      password: random 64
```

Comment:
Is there a reason we don't just use the same admin password for all three pieces?  Think of it as a _Prometheus_ UI password?